Projects/Profile/Sources/Views/Screens/DeleteAccount/DeleteRequestLoadingView.swift

```diff
@@ -99,20 +100,33 @@ struct DeleteRequestLoadingView: View {
     }
 
     private func sendSlackMessage(details: MemberDetails) {
-        let bot = SlackBot()
-        bot.postSlackMessage(memberDetails: details)
-            .onValue { status in
-                self.screenState = status ? .success : .error
-                if status {
+        self.octopus.client
+            .perform(mutation: OctopusGraphQL.MemberDeletionRequestMutation())
+            .onValue { value in
+                if let _ = value.memberDeletionRequest?.message {
```

Comment:
What happens with message that we get from backend? I think we want to display what we get there. @sladan-hedvig

Reply:
Yes, then just edit state error that it accept string